dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/ScanContext.java

```diff
@@ -79,11 +94,20 @@ void endWalkDir(FileStatus dir, long nFiles, long nDirs, long accumFileSize) thr
       this.numDirs += nDirs;
       this.accumulatedFileSize += accumFileSize;
 
+      String absolutePath = dir.getPath().toUri().getPath();
```

Comment:
Is `getPath().toUri().getPath()` intentional?

Reply:
Absolutely,
it gets rid of protocol, host and port and leaves path component only